_data/projects/packer.yml

```diff
@@ -0,0 +1,11 @@
+name: packer
+desc: A open source package manager that can run on linux systems
+site: https://github.com/TinyRobby/packer
+tags:
+- c++
+- linux
+- packages
+- apt
+upforgrabs:
+  name: upforgrabs
+  link: https://github.com/TinyRobby/packer/discussions/2
```

Comment:
🤔 this should be a list of issues related to the project. Is that something you plan to setup?